hypha/apply/activity/adapters/emails.py

```diff
@@ -441,14 +471,27 @@ def reviewers(self, source):
             and not reviewer.is_apply_staff
         ]
 
+    def partners(self, source) -> List[str]:
+        """Get all partner emails from the source (if any)
+
+        Returns:
+            list: emails of all partners on the source
+        """
+        return [partner.email for partner in source.partners.all()]
```

Comment:
I did this via comprehension because line 432 needs a python list to append to: `recipients = [source.user.email]` but I think you're totally right in terms of this function being able to be removed. I also replaced the comprehension with `[*source.partners.values_list('email', flat=True)]`, not too sure which is better in terms speed & resources though

Reply:
to be able to add `source.user.email` to the recipient iterable it needs to be a pure list rather than a queryset but totally agreed on readability